allensdk/brain_observatory/vbn_2022/metadata_writer/lims_queries.py

```diff
@@ -0,0 +1,758 @@
+from typing import List, Tuple, Dict, Any, Optional
+import pandas as pd
+from allensdk.internal.api import PostgresQueryMixin
+
+from allensdk.internal.api.queries.lims_queries import (
```

Comment:
There are a few docstrings in this module don't match the signature. Please check them to make sure there are no typos/missing arguments in the docstring